fix identify unified integration when running create-id-set (#1239)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
* Fixed an issue where in some cases the `get_remote_file` function failed due to an invalid path.

> * Fixed an issue where running **create-id-set** and there are an integration yml with an empty value in the script field, it's identify it as a unified integration.

Reviewer: ```suggestion
* Fixed an issue where the **create-id-set** command did not identify unified integrations correctly.
```
